extensions/amp-story/1.0/amp-story-reaction-quiz.css

```diff
@@ -17,10 +17,28 @@
 @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
 
 .i-amphtml-story-reaction-quiz-container {
+    --reaction-prompt-text-color: white;
+    --reaction-option-text-color: #5F6368;
+    --reaction-background-color: white;
+    --reaction-theme-border: #DADCE0 !important;
+    --reaction-theme-shading: #ECEDEF !important;
+    --reaction-chip-radius: 32px !important;
+    --reaction-chip-shadow: none !important;
+    --reaction-answer-choice-border: var(--reaction-accent-color) !important;
+    --reaction-answer-choice-background: white !important;
     --correct-color: #5BBA74 !important;
     --correct-color-shaded: #00600f !important;
     --incorrect-color: #F34E4E !important;
     --incorrect-color-shaded: #B71C1C !important;
```

Comment:
These are specific to quizzes so I moved them here

Reply:
I think we'll want polls to support most of these as well. This is fine for now, but I filed https://github.com/ampproject/amphtml/issues/28530

Reply:
Actually these properties are specific to quizzes in the sense that polls don't have borders or shading, or any answer choices being right or wrong. In that sense, the only two properties that can be reused are the background color and accent color that are already being shared from amp-story-reaction.css. I updated the issue to have all the details of the shared variables just in case